pkg/cmd/roachprod/install/cluster_synced.go

```diff
@@ -683,9 +684,11 @@ on_exit() {
 }
 trap on_exit EXIT
 [[ -f ~/.ssh/authorized_keys ]] && cat ~/.ssh/authorized_keys > "${tmp1}"
+echo "${keys_data}" >> "${tmp1}"
 sort -u < "${tmp1}" > "${tmp2}"
-mv "${tmp2}" ~/.ssh/authorized_keys
-`
+sudo install --mode 0600 --owner ` + sharedUser +
+				` --group ` + sharedUser +
+				`"${tmp2}" ~` + sharedUser + `/.ssh/authorized_keys`
```

Comment:
I think we need a space before `"${tmp2}"`.

Reply:
Good catch, sorry for the trouble.

https://github.com/cockroachdb/cockroach/pull/36889